packages/ui/package.json

```diff
@@ -35,29 +30,13 @@
   },
   "publishConfig": {
     "exports": {
-      "./components/ui/*": {
-        "default": "./dist/components/ui/*.js",
-        "types": "./dist/components/ui/*.d.ts"
-      },
-      "./components/ui/dutch-auction-component": {
-        "default": "./dist/components/ui/dutch-auction-component/index.js",
-        "types": "./dist/components/ui/dutch-auction-component/index.d.ts"
-      },
-      "./components/ui/identicon": {
-        "default": "./dist/components/ui/identicon/index.js",
-        "types": "./dist/components/ui/identicon/index.d.ts"
+      "./components/ui/icons/*": {
+        "default": "./dist/components/ui/icons/*.js",
+        "types": "./dist/components/ui/icons/*.d.ts"
       },
-      "./components/ui/slider": {
-        "default": "./dist/components/ui/slider/index.js",
-        "types": "./dist/components/ui/slider/index.d.ts"
-      },
-      "./components/ui/tx/view/asset-icon": {
-        "default": "./dist/components/ui/tx/view/asset-icon/index.js",
-        "types": "./dist/components/ui/tx/view/asset-icon/index.d.ts"
-      },
-      "./components/ui/tx/view/value": {
-        "default": "./dist/components/ui/tx/view/value/index.js",
-        "types": "./dist/components/ui/tx/view/value/index.d.ts"
+      "./components/ui/*": {
+        "default": "./dist/components/ui/*/index.js",
+        "types": "./dist/components/ui/*/index.d.ts"
       },
```

Comment:
Now all components have their own directory and an `index.tsx` file – all imports should look like `import {} from '@penumbra-zone/ui/components/ui/component-name`.

I really wish to remove the `components/ui` part of the import but it would make the PR even bigger.